icons/matplotlib/matplotlib-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
4.617 1.378.743 2.23.124.842.153 1.732.083 2.67zm-3.726.255c.04-.54.036-1.06-.012-1.562a4.072 4.072 0 0 0-.313-1.335 2.24 2.24 0 0 0-.713-.923c-.295-.237-.684-.355-1.167-.355-.237 0-.471.033-.703.099-.232.066-.477.18-.735.341-.249.161-.51.369-.785.625s-.568.573-.88.952l-.313 4.204c.454.644.895 1.136 1.325 1.477.439.331.91.497 1.412.497.464 0 .866-.118 1.205-.355.34-.237.628-.544.864-.923.246-.379.434-.81.565-1.292.129-.474.213-.96.25-1.45z"/><path fill="#11557c" d="M79.3 74.538c-2.761 0-5.357-1.075-7.31-3.028s-3.028-4.549-3.028-7.31c0-2.761 1.075-5.357 3.028-7.31s4.549-3.028 7.31-3.028 5.357 1.075 7.31 3.028 3.028 4.549 3.028 7.31-1.075 5.357-3.028 7.31-4.549 3.028-7.31 3.028zm0-20.372c-2.68 0-5.2 1.044-7.095 2.939a9.963 9.963 0 0 0-2.939 7.095c0 2.68 1.044 5.2 2.939 7.095a9.969 9.969 0 0 0 7.095 2.939c2.68 0 5.2-1.044 7.095-2.939a9.969 9.969 0 0 0 2.939-7.095c0-2.68-1.044-5.2-2.939-7.095a9.963 9.963 0 0 0-7.095-2.939z"/><path fill="#11557c" d="M79.3 72.263c-4.445 0-8.062-3.617-8.062-8.062 0-4.446 3.617-8.063 8.062-8.063s8.062 3.617 8.062 8.063c0 4.445-3.617 8.062-8.062 8.062zm0-16.009c-4.381 0-7.945 3.564-7.945 7.946 0 4.381 3.564 7.945 7.945 7.945s7.945-3.564 7.945-7.945-3.564-7.946-7.945-7.946z"/><path fill="#11557c" d="M79.3 69.928c-3.158 0-5.728-2.569-5.728-5.728s2.569-5.728 5.728-5.728 5.728 2.57 5.728 5.728-2.57 5.728-5.728 5.728zm0-11.339c-3.094 0-5.61 2.517-5.61 5.611s2.517 5.61 5.61 5.61 5.61-2.517 5.61-5.61-2.516-5.611-5.61-5.611z"/><path fill="#11557c" d="M79.3 67.594c-1.871 0-3.394-1.522-3.394-3.394s1.522-3.394 3.394-3.394 3.394 1.522 3.394 3.394-1.523 3.394-3.394 3.394zm0-6.67c-1.807 0-3.276 1.47-3.276 3.276s1.47 3.276 3.276 3.276 3.276-1.47 3.276-3.276-1.47-3.276-3.276-3.276z"/><path fill="#11557c" d="M79.3 65.426c-.676 0-1.226-.55-1.226-1.226s.55-1.226 1.226-1.226 1.226.55 1.226 1.226-.55 1.226-1.226 1.226zm0-2.334a1.11 1.11 0 0 0 0 2.216 1.11 1.11 0 0 0 0-2.216z"/><path fill="#11557c" d="m86.48 69.676-6.984-5.443h2.063a.068.068 0 0 0 .069-.068c0-.245-.038-.488-.114-.721a.07.07 0 0 0-.087-.044l-1.934.628 4.091-5.13a.069.069 0 0 0-.011-.097 6.834 6.834 0 0 0-1.867-1.059l-.002-.001a.067.067 0 0 0-.087.041l-2.304 6.141-1.964-8.602a.07.07 0 0 0-.083-.052 9.142 9.142 0 0 0-3.817 1.891l-.001.001a.068.068 0 0 0-.008.096l5.599 6.707-6.851-3.299a.07.07 0 0 0-.092.032 8.005 8.005 0 0 0-.663 4.895.069.069 0 0 0 .068.056l.012-.001 7.296-1.324-3.621 1.743a.07.07 0 0 0-.035.039.07.07 0 0 0 .003.053c.104.217.226.425.363.623.01.015.026.026.044.029l.012.001a.074.074 0 0 0 .041-.013l3.572-2.475-1.219 5.342a.07.07 0 0 0 .052.083 5.753 5.753 0 0 0 2.234.062.07.07 0 0 0 .056-.08l-.903-5.315 5.479 6.871a.07.07 0 0 0 .097.012 9.104 9.104 0 0 0 1.508-1.525.07.07 0 0 0-.012-.097z"/><path fill="#11557c" d="M79.358 74.43h-.117V64.342l-7.133 7.133-.083-.083.042-.041 7.091-7.092H69.07v-.117h10.089l-7.134-7.134.083-.083 7.133 7.133V53.97h.117v10.089l7.134-7.134.083.083-.042.041-7.092 7.093h10.088v.117H79.442l7.133 7.133-.083.083-.041-.042-7.093-7.092z"/></svg>
```

Comment:
Still multiple paths here. You can try using [SVGOMG](https://jakearchibald.github.io/svgomg/) or this SVG optimizer -> https://www.ofoct.com/image-converter/svg-optimizer.html. 
Both websites have a `merge Paths` option

Reply:
I do not understand why the fact the SVG as multiple paths is an issue, lots of SVG have multiple paths, even for `plain` version of icon / logo.

I don't think reducing to one path will improve the file size a lot, and it will complicate things if changes need to be made on the SVG afterwards.

And for info, for these SVG files, even using the links you shared you won't be able to reduce the number of path since these paths cannot be merge (at least with my knowledge and understanding of the SVG specification).

Last point, **icomoon** when importing SVGs will convert them to font glyph so the number of paths doesn't matter.

Reply:
I'm not sure what the exact reasoning for joining paths is either, but I am trying to enforce it because it is a part of the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)
> The plain and line versions (with or without wordmark) need to stay as simple as possible. They must have only one color and the paths are united.

Maybe @Panquesito7 could tell us more about why that is, so we can update the wiki with more info (or remove it if it's not needed).


As for merging the paths, you're right. Doesn't work because the last path starts with `m` instead of `M`. It should still be possible in Inkscape though. I gave it a try and almost got there with this
![matplotlib svg](https://user-images.githubusercontent.com/43886029/181615952-e1af2521-033e-4430-a2fb-8cabaf1c4900.svg)

almost identical, but the lines from the radar circle thingy are cutting through in some places

Here's both after eachother for comparison
![matplotlib](https://user-images.githubusercontent.com/43886029/181616082-b017349a-d127-4bf4-9d11-595f6f528535.svg)
![matplotlib svg](https://user-images.githubusercontent.com/43886029/181616093-38ac6346-4e21-4a27-9eb5-b7c4d173be54.svg)

Reply:
Yes the result of the "merge" give incorrect result and like I said the file size won't decrease, in fact the version you get is 3KB **(25%)** heavier than the one I created (with multiple paths)

Reply:
Having few / fewer paths is, lot of the time a good idea, **but** wanting to optimize more than what you can get with svg optimizers (like [SVG Viewer](https://www.svgviewer.dev/) or [SVGOMG](https://jakearchibald.github.io/svgomg/)) won't give you the expected result or will create unstable files

Reply:
@BenSouchet Alright, I think I got the paths merged so that it's identical now:
## New
![image](https://user-images.githubusercontent.com/43886029/181618253-ebe4e8b1-a62b-4a28-a7fb-406a13a772ca.svg)

## Old
![matplotlib](https://user-images.githubusercontent.com/43886029/181618240-e25623eb-20d7-459a-98bc-a72074a3f0b0.svg)


## New & Old side by side
<img align="left" src="https://user-images.githubusercontent.com/43886029/181617663-470424db-cc5b-4db9-a022-f67695011823.svg" width="200" />
<img align="left" src="https://user-images.githubusercontent.com/43886029/181618132-66fc9291-f90c-4162-8503-867ac73021db.svg" width="200" />

Reply:
> Having few / fewer paths is, lot of the time a good idea, **but** wanting to optimize more than what you can get with svg optimizers (like [SVG Viewer](https://www.svgviewer.dev/) or [SVGOMG](https://jakearchibald.github.io/svgomg/)) won't give you the expected result or will create unstable files

I don't have the knowledge to agree or disagree with you here, but your argument seems logical, so I'm just gonna wait and see what the other @devicons/maintainer s think about this one

Reply:
@Snailedlt Nice, yep it's seems they are identical
 
I compared file size of the one you just created and it's heavier (**16%**), so don't know also what is better but SVG I created for this project had rendering issue and like I said when **icomoon** convert the SVG the result is a glyph so I think the two version, mine and yours will give the exact same result.

So at the end, the SVG with only one path that is heavier and impossible to edit again (since every shape is merged) is maybe not the best one.

Yeah I'm also interested to see what the others maintainers think on that topic :)

Reply:
Yeah rom what I can tell multiple paths is better in this instance, but I'm hesitant to merge because of the SVG Standards like I mentioned earlier.

I guess we'll have to wait and see for the other maintainers